examples/pixieExample/README.md

```diff
@@ -0,0 +1,183 @@
+# Collect custom Pixie metrics from your k8s cluster and push to New Relic for Alerting and Dashboarding
+
+The following documentation explains how to configure the New Relic FLEX integration to collect custom Pixie metrics from your k8s cluster.
+
+Currently, for security reasons, when [Pixie](https://pixielabs.ai) is installed into your k8s cluster, New Relic only fetches and stores data that related to an application's performance. Therefore, you can only create Alert and Dashboard on a predefined subset of the data collected by Pixie [read more here](https://docs.newrelic.com/docs/kubernetes-pixie/auto-telemetry-pixie/pixie-data-security-overview/).
+
+The persisted performance metrics include, but are not limited to:
+
+-   Golden metrics (throughput, latency, error rate) for HTTP-based services
+-   HTTP transaction data
+-   Database transaction data (for MySQL & PostgreSQL)
+-   Distributed tracing
+-   JVM metrics
+
+With New Relic Flex, you now can report and alert on all other available Pixie metrics by:
+
+-   write custom [Pixie script](https://docs.pixielabs.ai/tutorials/pxl-scripts/write-pxl-scripts/)
```

Comment:
We should be using sentence case for these dot points. So just use a capital at the start of each bullet point, can see there are a few of these below.

Reply:
done